docs/querying-data.rst

```diff
@@ -0,0 +1,196 @@
+Querying data
+=============
+
+Before you begin, read the :doc:`index` page to learn about how data is stored in Kingfisher Views.
+
+Tables in Kingfisher Views can contain summaries of:
+
+* individual OCDS releases
+* individual OCDS releases from an OCDS record
+* compiled OCDS releases generated by Kingfisher Process
+* compiled OCDS releases published as part of an OCDS record
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled releases to begin with.
+
+List all schemas in the database
+--------------------------------
+
+Kingfisher Views creates database schemes to store summary data.
+
+The following query returns a list of schemas in the database:
+
+.. code-block:: sql
+
+  select
+    schema_name
+  from
+    information_schema.schemata;
+
+Set the schema to query
+-----------------------
+
+The following command sets the current schema to ``view_data_collection_1257_1259`` by making it the first item in the search path:
+
+.. code-block:: sql
+
+  set
+    search_path
+  to
+    view_data_collection_1257_1259, public;
+
+.. note::
+
+  Depending on the tool used to query the database, you may need to include the above command at the start of each query you run.
+
+List the collections in the current schema
+------------------------------------------
+
+The following query lists the collections in the current schema, with the name of the data source and the type of data summarised:
+
+.. code-block:: sql
+
+  select distinct
+      collection_id,
+      source_id,
+      release_type
+  from
+      release_summary
+  join
+      collection
+  on
+      release_summary.collection_id = collection.id
+  order by
+      collection_id desc;
+
+The ``release_type`` column indicates the type of data stored in the collection:
+
+* ``release`` identifies individual releases
+* ``compiled_release`` identifies compiled releases generated by Kingfisher Process
+* ``record`` identifies compiled releases published as part of an OCDS record
+
```

Comment:
This is good and clear.

Normally I'd say we should be including an example of the output, but this is quite a simple query and the columns are explained clearly at the end of the section.